tests/core/secrets_collection_test.py

```diff
@@ -133,7 +133,9 @@ def test_unicode_decode_error(self, mock_log):
             logic.scan_file('filename')
 
         assert mock_log.info_messages == 'Checking file: filename\n'
-        assert mock_log.warning_messages == 'filename failed to load.\n'
+        assert mock_log.warning_messages == 'filename failed to load and could not be scanned.\n' \
```

Comment:
The error used by this test is defined here: 
https://github.com/IBM/detect-secrets/blob/master/tests/core/secrets_collection_test.py#L502

![Screenshot 2023-03-17 at 4 30 03 PM](https://user-images.githubusercontent.com/12804086/226057057-c60a4fbe-36f2-435a-a440-24bb6c2ec2df.png)